starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,66 @@
+body{
+  font-family: 'Open Sans', sans-serif;
+  font-size: 1em;
+}
+header {
+  background-image: url("../images/apple-pie.jpg");
+  background-size: cover;
+  height: 500px;
+}
+header h1{
+  color:rgb(255,255,255);
+  text-align: center;
+  padding-top: 250px;
```

Comment:
Bien apañado, es una solución válida siempre y cuando sepamos el height del header! Más adelante veréis que flexbox es clave para centrar verticalmente y horizontalmente elementos dentro de cajas que puedan variar de tamaño. Os lo acepto! ;)